src/routes/CreateSafePage/components/SafeCreationProcess.tsx

```diff
@@ -212,6 +224,8 @@ function SafeCreationProcess(): ReactElement {
     })
   }
 
+  const explorerUrl = newSafeAddress ? getExplorerInfo(newSafeAddress) : undefined
```

Comment:
If this is only used for the modal, I would suggest changing it to depend on `newSafeAddress` and move it into the props directly.

Reply:
Do you mean
```
const explorerUrl = (safeAddress: string) => (safeAddress ? getExplorerInfo(safeAddress) : undefined)
```

and call the method in the `InlineEthHashInfo` props?

Reply:
No, render the new modal conditionally like so:

```jsx
{showCouldNotLoadModal && newSafeAddress && <Modal ... >}
```